### PR TITLE
Embed BPF templates in cilium-agent

### PR DESCRIPTION
**Pull request by tgraf**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/cilium/cilium/pull/354%23discussion_r106910811%22%2C%20%22https%3A//github.com/cilium/cilium/pull/354%23discussion_r106914461%22%2C%20%22https%3A//github.com/cilium/cilium/pull/354%23discussion_r106915828%22%2C%20%22https%3A//github.com/cilium/cilium/pull/354%23discussion_r106916820%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%202d61a2c601f642a3a3166f5206e24a6735310c07%20.travis/prepare.sh%204%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/cilium/cilium/pull/354%23discussion_r106910811%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Why%20doesn%27t%20this%20belong%20to%20vendor%3F%22%2C%20%22created_at%22%3A%20%222017-03-20T14%3A16%3A32Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/5714066%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/aanm%22%7D%7D%2C%20%7B%22body%22%3A%20%22It%27s%20a%20binary%2C%20we%20only%20need%20it%20at%20development%20time.%22%2C%20%22created_at%22%3A%20%222017-03-20T14%3A29%3A58Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/1417913%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tgraf%22%7D%7D%2C%20%7B%22body%22%3A%20%22Can%20you%20document%20that%20somewhere%3F%20We%20already%20need%20%60go-swagger%60%20and%20now%20this%20%60go-bindata%60%20as%20well%22%2C%20%22created_at%22%3A%20%222017-03-20T14%3A34%3A37Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/5714066%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/aanm%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%20will%20add%20this%20to%20the%20developer%20docs.%22%2C%20%22created_at%22%3A%20%222017-03-20T14%3A38%3A05Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/1417913%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tgraf%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20.travis/prepare.sh%3AL11-18%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull 2d61a2c601f642a3a3166f5206e24a6735310c07 .travis/prepare.sh 4'></a> &nbsp;<b><a href='https://github.com/cilium/cilium/pull/354#discussion_r106910811'>File: .travis/prepare.sh:L11-18</a></b>
- <a href='https://github.com/aanm'></a> Why doesn't this belong to vendor?
- <a href='https://github.com/tgraf'></a> It's a binary, we only need it at development time.
- <a href='https://github.com/aanm'></a> Can you document that somewhere? We already need `go-swagger` and now this `go-bindata` as well
- <a href='https://github.com/tgraf'></a> I will add this to the developer docs.

<a href='https://www.codereviewhub.com/cilium/cilium/pull/354?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/cilium/cilium/pull/354?approve=1'></a>&nbsp;<a href='https://github.com/cilium/cilium/pull/354'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by aanm** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/cilium/cilium/pull/354'>Refresh</a>.<a href='#crh-update-none'></a>

File: .travis/prepare.sh [hunk 1]
```diff
@@ -11,5 +11,7 @@ sudo tar -C /usr/local -xJf $CLANG_FILE
 sudo ln -s /usr/local/$CLANG_DIR /usr/local/clang
 rm $CLANG_FILE
 
+go get -u github.com/jteeuwen/go-bindata/...
```
Comment (aanm) [inline .travis/prepare.sh:14]: Why doesn't this belong to vendor?

Comment (tgraf) [reply .travis/prepare.sh:14]: It's a binary, we only need it at development time.

Comment (aanm) [reply .travis/prepare.sh:14]: Can you document that somewhere? We already need `go-swagger` and now this `go-bindata` as well

Comment (tgraf) [reply .travis/prepare.sh:14]: I will add this to the developer docs.

**Review by aanm — APPROVED** [review]

**Pull request merged**